action.yml

```
@@ -95,7 +95,9 @@ runs:
       run: |
         # For debug
         # sudo env "PATH=$PATH" uv pip install --system --break-system-packages git+https://github.com/ultralytics/actions@main codespell tomli
-        packages="ultralytics-actions"
+        sudo env "PATH=$PATH" uv pip install --system --break-system-packages git+https://github.com/ultralytics/actions@fix-indent codespell tomli
```

❗ **CRITICAL**: This line force-installs from an unpinned Git branch (`fix-indent`) with elevated privileges (`sudo`) and `--break-system-packages`, bypassing the controlled `packages` flow. This creates non-reproducible builds, broadens the attack surface, and alters behavior by installing tools unconditionally. It also risks conflicts with subsequent package handling. Recommend disabling this debug/install line or gating it behind an input and using a pinned release/commit.

**Suggested change:**
```suggestion
        # sudo env "PATH=$PATH" uv pip install --system --break-system-packages git+https://github.com/ultralytics/actions@fix-indent codespell tomli
```